core/frameworks/pixel/components/alert-banner.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Frameworks.Pixel.AlertBanner do
+  use CoreWeb, :pixel
+
+  @types ~w(success warning error info)a
+
+  @bg_classes %{
+    success: "bg-successlight",
+    warning: "bg-warninglight",
+    error: "bg-errorlight",
+    info: "bg-primarylight"
+  }
+
+  @text_classes %{
+    success: "text-success",
+    warning: "text-warning",
+    error: "text-error",
+    info: "text-primary"
+  }
+
+  attr(:type, :atom, default: :info, values: @types)
+  attr(:message, :string, default: nil)
+  attr(:class, :string, default: "")
+  slot(:inner_block)
+
+  def alert(assigns) do
+    assigns =
+      assigns
+      |> assign_new(:message, fn -> nil end)
+      |> assign(:bg_class, Map.fetch!(@bg_classes, assigns.type))
+      |> assign(:text_class, Map.fetch!(@text_classes, assigns.type))
+
+    ~H"""
+    <div
+      class={[
+        "rounded flex font-semibold p-[0.625rem_1rem] justify-center items-center gap-[0.625rem]",
```

Comment:
Consider to use one of the custom fonts defined in tailwind. Or add a new one there if this is a new font/text variant